.github/workflows/refresh_db.yml

```diff
@@ -2,8 +2,8 @@ name: DB Refresh (Objects API)
 
 on:
   schedule:
-    - cron: '0 5 * * *'  # every day 5am GMT (12-1am EST/EDT)
-#    - cron: '*/20 * * * *'  # every 20 minutes
+#    - cron: '0 5 * * *'  # every day 5am GMT (12-1am EST/EDT)
+    - cron: '*/20 * * * *'  # every 20 minutes
```

Comment:
@Sigfried @Leskena2 I decided to set the refresh rate to to once every 20 minutes (not once every 2 hours as we talked about today, nor 1x/day as it was previously), for a few reasons:
1. GitHub doesn't seem to care if we run this often.
2. It exits if no data is found, without remaking derived tables or anything.
3. We've set up a process to where if the user does a refresh while it's running, it is not a problem.
4. If all works smoothly, this is better UX, as users will less frequently need to manually refresh and wait.